_projects/2022/sunpy/theme.md

```diff
@@ -13,8 +13,8 @@ mentors:
 initiatives:
  - GSOC
 project_size:
- - medium
- - large
+ - 175 h
```

Comment:
Dont we have to pick?

Reply:
I'm not so sure, ideally we should or have two separate descriptions of what is expected in one or the other.

Reply:
My understanding is that it's the student's proposal that picks. IMO if they write a compelling proposal to fill the 350hr with the idea and the mentors agree it could end up high on the pick list.